pages/services/kubernetes/1.0.0-1.9.3/upgrade/index.md

```diff
@@ -0,0 +1,167 @@
+---
+layout: layout.pug
+navigationTitle: Upgrade
+title: Upgrade
+menuWeight: 50
+excerpt:
+---
+
+<!-- This source repo for this topic is https://github.com/mesosphere/dcos-kubernetes -->
+
+
+## Before you start
+
+Currently, updating the package may trigger an update of a subset or
+all the components managed by this framework, depending on whether new versions
+of these components or related ones are part of the new release.
+
+Updating these components means the Kubernetes cluster may incur in some
```

Comment:
Updating these components means the Kubernetes cluster may expereince some
downtime or, in the worst-case scenario, cease to function properly. And while
we are committed to make the user experience as robust and smooth as possible, there is no such thing as fail-proof software.

Reply:
Done.